libkbfs/tlf_journal.go

```diff
@@ -543,17 +572,20 @@ func (j *tlfJournal) flushOneMDOp(ctx context.Context) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if mdID == (MdID{}) {
+	// TODO: Doing the revision number check here is inefficient,
```

Comment:
As I mentioned above, I think you should fix this now -- not for efficiency, but for code symmetry.